integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'name': u'Mahendra Rajmarg',
+                'name:en': u'Mahendra Highway',
+                'name:ne': u'महेन्द्र राज्मार्ग्',
+                'name:pl': u'Mahendra',
+                'network': u'np:national',
+                'ref': u'H01',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q3595565',
+                'wikipedia': u'en:Mahendra Highway',
+            }, ways=[27030698]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 27030698,
+                'network': u'NP:national',
+            })
+
+    def test_npregional(self):
+        import dsl
+
+        z, x, y = (16, 48796, 27744)
+
+        self.generate_fixtures(
+            dsl.is_in('NP', z, x, y),
+            # https://www.openstreetmap.org/way/352615958
+            dsl.way(352615958, dsl.tile_diagonal(z, x, y), {
+                'highway': u'primary',
+                'name': u'Birtamod — Chandragadhi',
+                'ref': u'F1',
+                'source': u'openstreetmap.org',
+                'surface': u'paved',
+            }),
+            dsl.relation(1, {
+                'distance': u'12.53 km',
+                'name': u'Birtamod — Chandragadhi',
+                'network': u'np:regional',
+                'ref': u'F1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[352615958]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 352615958,
+                'network': u'NP:regional',
+            })
+
+    def test_phnational(self):
+        import dsl
+
+        z, x, y = (16, 55524, 30719)
+
+        self.generate_fixtures(
+            dsl.is_in('PH', z, x, y),
+            # https://www.openstreetmap.org/way/460910438
+            dsl.way(460910438, dsl.tile_diagonal(z, x, y), {
+                'highway': u'trunk',
+                'int_ref': u'AH26',
+                'lanes': u'2',
+                'name': u'Maharlika Highway',
+                'nat_name': u'Daang Maharlika (LT)',
+                'official_name': u'Daang Maharlika Highway',
+                'ref': u'1;AH26',
+                'source': u'openstreetmap.org',
+                'surface': u'concrete',
+            }),
+            dsl.relation(1, {
+                'name': u'N1',
+                'network': u'ph:nhn',
+                'nhn:class': u'primary',
+                'ref': u'1',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[460910438]),
+            dsl.relation(2, {
+                'int_ref': u'AH26',
+                'int_ref:colour': u'blue',
+                'name': u'National Highway',
+                'name:en': u'National Highway',
+       
```

Comment:
Are M prefixed refs generally national `M-road` and this one just happens to fall in Capetown? Are the Capetown motorways signed different than other admin-1 regions in South Africa?

Reply:
Although the network was given as `za:capetown`, the [numbering system of South Africa](https://en.wikipedia.org/wiki/Numbered_routes_in_South_Africa) calls these "metropolitan". Apparently, they're present in a lot of cities. Since we're using this for shields, it doesn't matter what city they're in (shields seem to be the same), so I've mapped it to `ZA:metropolitan`. That, and other, ZA fixes in d7cb3aa.